cypress/e2e/invoices/approve-invoice.cy.js

```diff
@@ -0,0 +1,81 @@
+import permissions from '../../support/dictionary/permissions';
+import testType from '../../support/dictionary/testTypes';
+import devTeams from '../../support/dictionary/devTeams';
+import TopMenu from '../../support/fragments/topMenu';
+import NewInvoice from '../../support/fragments/invoices/newInvoice';
+import NewInvoiceLine from '../../support/fragments/invoices/newInvoiceLine';
+import Invoices from '../../support/fragments/invoices/invoices';
+import VendorAddress from '../../support/fragments/invoices/vendorAddress';
+import NewFund from '../../support/fragments/finance/funds/newFund';
+import Funds from '../../support/fragments/finance/funds/funds';
+import DateTools from '../../support/utils/dateTools';
+import Helper from '../../support/fragments/finance/financeHelper';
+import Transaction from '../../support/fragments/finance/fabrics/newTransaction';
+import Organizations from '../../support/fragments/organizations/organizations';
+import SettingsMenu from '../../support/fragments/settingsMenu';
+import SettingsInvoices from '../../support/fragments/invoices/settingsInvoices';
+import Users from '../../support/fragments/users/users';
+
+describe('ui-invoices: Approve invoice', () => {
+  const invoice = { ...NewInvoice.defaultUiInvoice };
+  const vendorPrimaryAddress = { ...VendorAddress.vendorAddress };
+  const invoiceLine = { ...NewInvoiceLine.defaultUiInvoiceLine };
+  const defaultFund = { ...NewFund.defaultFund };
+  const subtotalValue = 100;
+  let user;
+
+  before(() => {
+    cy.getAdminToken();
+    cy.loginAsAdmin({ path:SettingsMenu.invoiceApprovalsPath, waiter: SettingsInvoices.waitApprovalsLoading });
+    SettingsInvoices.checkApproveAndPayCheckboxIsDisabled();
+    Organizations.getOrganizationViaApi({ query: `name=${invoice.vendorName}` })
+      .then(organization => {
+        invoice.accountingCode = organization.erpCode;
+        console.log(organization);
+        Object.assign(vendorPrimaryAddress,
+          organization.addresses.find(address => address.isPrimary === true));
+          cy.getBatchGroups()
+          .then(batchGroup => { 
+            invoice.batchGroup = batchGroup.name;
+            Funds.createFundViaUI(defaultFund)
```

Comment:
please add function for deleting created fund

Reply:
If we have transactions we can't delete funds and invoices